nsdb-common/src/test/resources/nsdb-test.conf

```diff
@@ -0,0 +1,134 @@
+# Copyright 2018 Radicalbit S.r.l.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+nsdb {
+
+  node {
+    hostname = "127.0.0.1"
+    port = 2552
+  }
+
+  grpc {
+    interface = "0.0.0.0"
+    interface = ${?GRPC_INTERFACE}
```

Comment:
Do we need env var in test confi file?

Reply:
No, we do not. It was a too lazy copy & paste